uaclient/cli/cves.py

```diff
@@ -0,0 +1,143 @@
+        return messages.TxtColor.INFOBLUE + priority + messages.TxtColor.ENDC
+    elif priority == "medium":
+        return (
+            messages.TxtColor.WARNINGYELLOW + priority + messages.TxtColor.ENDC
+        )
+    elif priority == "high":
+        return messages.TxtColor.ORANGE + priority + messages.TxtColor.ENDC
+    elif priority == "critical":
+        return messages.TxtColor.FAIL + priority + messages.TxtColor.ENDC
+    else:
+        return priority
+
+
+def _get_cve_table_rows(cve_vulnerabilities):
+    rows = []
+
+    for package_name, package_info in cve_vulnerabilities.packages.items():
+        for cve in package_info.cves:
+            cve_info = cve_vulnerabilities.cves.get(cve.name)
+
+            if cve_info:
+                rows.append(
+                    (
+                        package_name,
+                        cve_info.priority,
+                        cve.fix_origin or "-",
+                        create_link(
+                            text=cve.name,
+                            url="https://ubuntu.com/security/{}".format(
+                                cve.name
+                            ),
+                        ),
+                    )
+                )
+
+    return rows
+
+
+def _format_cve_rows(cve_rows):
+    formatted_rows = []
+    for row in sorted(
+        cve_rows,
+        key=lambda row: (
+            row[0],
+            ("critical", "high", "medium", "low", "negligible").index(row[1]),
+        ),
+    ):
+        formatted_rows.append(
+            (row[0], _colorize_priority(row[1]), row[2], row[3])
+        )
+
+    return formatted_rows
+
+
+def _list_cves(args, cfg: UAConfig):
+    cve_vulnerabilities = _get_cve_vulnerabilities(args, cfg=cfg)
+
+    if cve_vulnerabilities.packages:
+        rows = _format_cve_rows(_get_cve_table_rows(cve_vulnerabilities))
+
+        if rows:
+            print(
+                Table(
+                    headers=["Package", "Priority", "Origin", "Vulnerability"],
+                    rows=rows,
+                )
+            )
+        else:
+            if args.unfixable:
+                print(messages.CLI_UNFIXABLE_CVES_NOT_AFFECTED)
+            elif args.fixable:
+                print(messages.CLI_FIXABLE_CVES_NOT_AFFECTED)
+            else:
+                print(messages.CLI_CVES_NOT_AFFECTED)
+    else:
+        if args.unfixable:
+            print(messages.CLI_UNFIXABLE_CVES_NOT_AFFECTED)
+        elif args.fixable:
+            print(messages.CLI_FIXABLE_CVES_NOT_AFFECTED)
+        else:
+            print(messages.CLI_CVES_NOT_AFFECTED)
+
+
+def action_cves(args, *, cfg: UAConfig, **kwargs):
+    if args.unfixable and args.fixable:
+        raise exceptions.InvalidOptionCombination(
+            option1="unfixable", option2="fixable"
+        )
+
+    try:
+        _list_cves(args=args, cfg=cfg)
+    except BrokenPipeError:
+        sys.stderr.close()
+
+
+vulnerability_list_args_group = ProArgumentGroup(
```

Comment:
nit
```suggestion
cves_args_group = ProArgumentGroup(
```

Reply:
Alternatively, now that this argument group doesn't need to be reused, it can be defined inline below at `argument_groups=...`